<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCrosshairSetupDialog</class>
 <widget class="QDialog" name="QCrosshairSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crosshair Setup</string>
  </property>
  <widget class="QLabel" name="crosshairLineWidthLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>55</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="crosshairLineLengthSpinBox">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>85</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="showCenterCheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>115</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="text">
    <string>Show Center</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showLeftCheckBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>85</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="text">
    <string>Show Left</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="crosshairLineWidthSpinBox">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>55</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="crosshairLineLengthLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>85</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Length</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="centerOpacitySpinBox">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>85</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="crosshairOpacitySpinBox">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>115</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="showBottomCheckBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>55</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="text">
    <string>Show Bottom</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="y_offsetSpinBox">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>55</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::PlusMinus</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="showTopCheckBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>25</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="text">
    <string>Show Top</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="x_offsetSpinBox">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>25</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::PlusMinus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="y_offsetLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>55</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-Offset</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="centerOpacityLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Opacity</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="centerSizeSpinBox">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>55</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="x_offsetLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>25</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Offset</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="centerSizeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>55</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Size</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="crosshairOpacityLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>115</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Opacity</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="centerGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>136</height>
    </rect>
   </property>
   <property name="title">
    <string>Center</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="crosshairGroupBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>201</width>
     <height>136</height>
    </rect>
   </property>
   <property name="title">
    <string>Crosshair</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="showRightCheckBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>115</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="text">
    <string>Show Right</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="offsetGroupBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>141</width>
     <height>136</height>
    </rect>
   </property>
   <property name="title">
    <string>Offset</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <zorder>offsetGroupBox</zorder>
  <zorder>crosshairGroupBox</zorder>
  <zorder>centerGroupBox</zorder>
  <zorder>crosshairLineWidthLabel</zorder>
  <zorder>crosshairLineLengthSpinBox</zorder>
  <zorder>showCenterCheckBox</zorder>
  <zorder>showLeftCheckBox</zorder>
  <zorder>crosshairLineWidthSpinBox</zorder>
  <zorder>crosshairLineLengthLabel</zorder>
  <zorder>centerOpacitySpinBox</zorder>
  <zorder>crosshairOpacitySpinBox</zorder>
  <zorder>showBottomCheckBox</zorder>
  <zorder>y_offsetSpinBox</zorder>
  <zorder>showTopCheckBox</zorder>
  <zorder>x_offsetSpinBox</zorder>
  <zorder>y_offsetLabel</zorder>
  <zorder>centerOpacityLabel</zorder>
  <zorder>centerSizeSpinBox</zorder>
  <zorder>x_offsetLabel</zorder>
  <zorder>centerSizeLabel</zorder>
  <zorder>crosshairOpacityLabel</zorder>
  <zorder>showRightCheckBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
